src/plugins/plot/stackedPlot/pluginSpec.js

```diff
@@ -360,14 +360,16 @@ describe('the plugin', function () {
       expect(legend[0].innerHTML).toEqual('Test Object');
     });
 
-    it('Renders an expanded legend for every telemetry', () => {
+    it('Renders an expanded legend for every telemetry', async () => {
       let legendControl = element.querySelector(
         '.c-plot-legend__view-control.gl-plot-legend__view-control.c-disclosure-triangle'
       );
       const clickEvent = createMouseEvent('click');
 
       legendControl.dispatchEvent(clickEvent);
 
+      await nextTick();
```

Comment:
need to wait for next tick as vue is manipulating DOM

Reply:
as we use Vue more, the more the nextTick() hoops we will have to jump through in the unit tests. As a result I've been removing unit tests that are 100% covered by e2e tests here and there. If this test is stable like this then should be fine to keep it. Hopefully some day we can get actual component testing going here